vpdq/cpp/vpdq_match.py

```diff
@@ -0,0 +1,90 @@
+import subprocess
+import os
+import sys
+import argparse
+
+
+def get_argparse() -> argparse.ArgumentParser:
+    ap = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+
+    ap.add_argument(
+        "-f",
+        "--queryHashFolder",
+        metavar="QUERY_HASH_Folder_PATH",
+        help="Query Hashes Folder's Name",
+    )
+    ap.add_argument(
+        "-i",
+        "--targetHashFile",
+        metavar="TARGET_HASH_FILE_PATH",
+        help="Target Hash file path for comparing with the query hashes",
+    )
+
+    ap.add_argument(
+        "-t",
+        "--matchDistanceTolerance",
+        metavar="Matching_distanceTolerance",
+        help="The hamming distance tolerance of between two frames. If the hamming distance is bigger than the tolerance, it will be considered as unmatched",
+        default="31",
+    )
+    ap.add_argument(
+        "-q",
+        "--qualityTolerance",
+        metavar="Matching_qualityTolerance",
+        help="The quality tolerance of matching two frames. If either frames is below this quality level then they will not be compared",
+        default="50",
+    )
+    ap.add_argument(
+        "-v",
+        "--verbose",
+        metavar="Verbose",
+        help="If verbose, will print detailed information.",
+        default=False,
+    )
+    return ap
+
+
+def main():
+    ap = get_argparse()
+    args, unknownargs = ap.parse_known_args(sys.argv)
```

Comment:
blocking: Danger!!! You are silently ignoring any parse errors and still running the code! This is a very hard class of error to discover!